_pages/team.md

```diff
@@ -1,16 +1,58 @@
 ---
 permalink: /team/
 title: Team 4 All
+feature_row:
+  - image_path: https://projectvikram.github.io/restricted_assets/images/ac_photo.jpg
+    alt: "Abhishek Choudhary (avatar image)"
+    title: "Abhishek Choudhary"
+    excerpt: "Founding Architect"
+    url: "/team/#Abhishek-guriqbal-Choudhary"
+    btn_class: "btn--primary"
+    btn_label: "Profile"
+  - image_path: https://projectvikram.github.io/restricted_assets/images/dr_srija.jpg
+    alt: "Dr Srija Katta (avatar image)"
+    title: "Dr Srija Katta"
+    excerpt: "Technology Evangelist-Pharma & Convenor"
+    url: "/team/#dr-srija-katta"
+    btn_class: "btn--primary"
+    btn_label: "Profile"
 ---
 
-# Team The Project Vikram Journal
```

Comment:
This was part of a cross branch merge today! Thanks for correcting.

This should be captured as an issue to automatically handle corrections, and flag any cross branch merges in the future